core/src/main/scala/pureconfig/configurable/package.scala

```diff
@@ -0,0 +1,20 @@
+package pureconfig
+
+import scala.util.Try
+import java.time.{ LocalDate, LocalDateTime, LocalTime }
+import java.time.format.DateTimeFormatter
+
+package object configurable {
```

Comment:
I think the behavior of modules should mimic the one of core as much as possible, so yes, for me it should go in `pureconfig.joda.configurable`.